client/state/data-layer/wpcom/jetpack/settings/test/index.js

```diff
@@ -35,18 +35,7 @@ describe( 'requestJetpackOnboardingSettings()', () => {
 	};
 
 	test( 'should dispatch an action for GET HTTP request to save Jetpack Onboarding settings', () => {
```

Comment:
Let's alter the test names here - these are no longer an onboarding-specific tests (this applies for most of the rest of the onboarding tests in this file).

Also, I just realized this test name has been wrong all along - we don't `save` settings - we `request` or `retrieve` them.

Reply:
0350947